README.md

```diff
@@ -4,14 +4,24 @@
     <strong>
       A starter to kit for developing  REST API in typescript and express.
   </p>
+  
 <!-- PROJECT SHIELDS -->
+[![Licence](https://img.shields.io/github/license/Ileriayo/markdown-badges?style=for-the-badge)](./LICENSE)
```

Comment:
Please align them in center and add a hyperlink to the npm package  (https://www.npmjs.com/package/create-express-ts-api)

Reply:
done